keyboards/rgbkb/mun/mcuconf.h

```diff
@@ -39,4 +21,4 @@
 #define STM32_PWM_USE_TIM4 TRUE
```

Comment:
I think this can also be removed, as TIM4 isnt currently being used?

Reply:
I thought I needed this for the WS2812 driver

Reply:
No driver is specified for WS2812, from what  can see. So it defaults to bitbang, which doesn't use a timer.

Reply:
Has `#define WS2812_PWM_DRIVER PWMD3`, so I assume TIM4 is left over from some other tinkering.

Reply:
```
#define WS2812_PWM_DRIVER PWMD3
#define WS2812_PWM_CHANNEL 2
#define WS2812_PWM_PAL_MODE 2
#define WS2812_DMA_STREAM STM32_DMA1_STREAM3
#define WS2812_DMA_CHANNEL 3
```

From my understanding, that requires one of the timers to operate. I thought it was TIM4

Reply:
Either way I'll double check everything this weekend and can resolve any issues.

Reply:
I think that would be timer 3? But i could be wrong. 

But unless `WS2812_DRIVER = pwm` is set, then it won't use the pwm driver, and will default to the bitbang one. 

Did you mean to use the PWM driver for the ws2812?

Reply:
@drashna it's using pwm now, please see original pr here:
https://github.com/qmk/qmk_firmware/pull/13239/files#diff-0ea3f1d2d976dc67361be11cdc194440a7b54056abb950b257a00d750e07698eR25

Reply:
... i don't know how  messed the pwm bit

Reply:
You would get compile issues as PWMD4 was being used but not defined, if removing it was an issue. I guess my question was more why would it still need to be enabled when nothing is using it, rather than isolating that TIM3 is being used for ws2812.

Removing it after having had a 2nd look is definitely the right thing in regards to the code that currently exists on develop.